controls/datagrid/filtering/filtering-ui.md

```diff
@@ -8,20 +8,21 @@ slug: datagrid-filtering-ui
 
 # Filtering UI
 
-The [.NET MAUI DataGrid]({%slug datagrid-overview%}) supports filtering through the UI.
+The [.NET MAUI DataGrid]({%slug datagrid-overview%}) provides you with a built-in filtering functionality, which allows the user to filter the data by one or more columns by clicking on the filtering icon next to the header text.
 
 ![Filtering UI](../filtering/images/datagrid-filtering-ui.gif)
 
-> Filtering UI appears when clicking on the filtering icon on each column's header'.
+> Filtering UI appears when clicking on the filtering icon on each column's header.
 
+You can control whether the filtering UI is available to the end users for the whole DataGrid with the `UserFilterMode` property:
 
-The Filtering UI exposes the following properties:
+* `UserFilterMode`(`DataGridUserFilterMode`)&mdash;Defines whether the Filter Glyph is present in the header of the currently available filterable columns. The available options are `Auto`, `Enabled`, and `Disabled`. The default value of the `UserFilterMode` is `Auto`.
```

Comment:
Is the "Filter Glyph" the same as the "filtering icon"?

Reply:
Yes, it's the same, I'll make it filtering icon